app/experimenter/nimbus-ui/src/components/AppLayoutSidebarLocked/index.tsx

```diff
@@ -0,0 +1,159 @@
+import { ReactComponent as Clipboard } from "../../images/clipboard.svg";
+import { ReactComponent as BarChart } from "./bar-chart.svg";
+import LinkExternal from "../LinkExternal";
+
+export const RESULTS_LOADING_TEXT = "Checking results availability...";
+
+const getSidebarItems = (metrics: { [metric: string]: string }) => {
+  const keys: string[] = [];
+  const labels: string[] = [];
+
+  Object.keys(metrics!).map((metric) => {
+    keys.push(metric);
+    labels.push(metrics[metric]);
+  });
+
+  return { keys, labels };
+};
+
+type AppLayoutSidebarLockedProps = {
+  testid?: string;
+  children: React.ReactNode;
+  status?: StatusCheck;
+  analysis?: AnalysisData;
+  analysisLoadingInSidebar?: boolean;
+  analysisError?: Error;
+} & RouteComponentProps;
+
+export const AppLayoutSidebarLocked = ({
+  children,
+  testid = "AppLayoutSidebarLocked",
+  status,
+  analysis,
+  analysisLoadingInSidebar = false,
+  analysisError,
+}: AppLayoutSidebarLockedProps) => {
+  const { slug } = useParams();
+  const { keys, labels } = getSidebarItems(analysis.other_metrics);
+
+  return (
+    <Container fluid className="h-100vh" data-testid={testid}>
+      <Row className="h-md-100">
+        <Col
+          md="3"
+          lg="3"
+          xl="2"
+          className="bg-light pt-2 border-right shadow-sm"
+        >
+          <nav
+            data-testid="nav-sidebar"
+            id="scrolling-navbar"
+            className="navbar fixed-top"
+          >
+            <Nav className="flex-column font-weight-semibold" as="ul">
+              <LinkNav
+                storiesOf="pages/Home"
+                className="mb-3 small font-weight-bold"
+                textColor="text-secondary"
+              >
+                <ChevronLeft className="ml-n1" width="18" height="18" />
+                Experiments
+              </LinkNav>
+              <LinkNav
+                route={`${slug}/design`}
+                storiesOf={"pages/Design"}
+                testid={"nav-design"}
+              >
+                <Clipboard className="sidebar-icon" />
+                Design
+              </LinkNav>
+              {analysisAvailable(analysis) ? (
+                <>
+                  <LinkNav
+                    route={`${slug}/results`}
+                    storiesOf={"pages/Results"}
+                    testid={"nav-results"}
+                  >
+                    <BarChart className="sidebar-icon" />
+                    Results
+                  </LinkNav>
+                  <Scrollspy
+                    items={[
+                      "monitoring",
+                      "overview",
+                      "results_summary",
+                      "default",
+                    ].concat(keys)}
+                    className="text-dark"
+                    currentClassName="font-weight-bold text-primary"
+                  >
+                    <div>
+                      <a href="monitoring" style={{ color: "inherit" }}>
```

Comment:
Can we use a Bootstrap class or a custom class rather than an inline style? I don't think we're using inline styles anywhere else.